docs/DeveloperGuide.md

```diff
@@ -132,11 +155,36 @@ The `Model`:
 
 ![Structure of the Storage Component](images/StorageClassDiagram.png)
 
-**API** : [`Storage.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/storagePerson/Storage.java)
+<div markdown="span" class="alert alert-info"> 
+**:information_source: Note:** ABC refers to `Deliverable`, `Meeting` and `Person`.
+
+</div>
+
+**API** : 
+[`StorageDeliverable.java`](https://github.com/AY2021S1-CS2103T-F11-2/tp/tree/master/src/main/java/seedu/address/storage/deliverable/StorageDeliverable.java)
+[`StorageMeeting.java`](https://github.com/AY2021S1-CS2103T-F11-2/tp/tree/master/src/main/java/seedu/address/storage/meeting/StorageMeeting.java)
+[`StoragePerson.java`](https://github.com/AY2021S1-CS2103T-F11-2/tp/tree/master/src/main/java/seedu/address/storage/person/StoragePerson.java)
 
 The `Storage` component:
```

Comment:
I think no `` formatting for `Storage` component
Need to standardize this too @chrystalquek @gabztcr @shadowezz

Reply:
Removed accordingly